srcpkgs/wluma/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'wluma'
+pkgname=wluma
+version=1.2.2
+revision=3
+archs="x86_64"
+#wrksrc=
+#create_wrksrc=yes
+build_style=gnu-makefile
+make_cmd="make build"
+hostmakedepends="pkg-config meson ninja"
+makedepends="Vulkan-Headers wayland-devel Vulkan-Tools"
+depends="wayland vulkan-loader mesa-vulkan-intel mesa-vulkan-radeon"
```

Comment:
`wayland` and `vulkan-loader` are detected automatically:

```
=> wluma-1.2.2_3: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libvulkan.so.1 <-> vulkan-loader>=1.0.57.0_1
   SONAME: libm.so.6 <-> glibc>=2.32_1
   SONAME: libwayland-client.so.0 <-> wayland>=1.10.0_1
   SONAME: libc.so.6 <-> glibc>=2.32_1
```

And I don't think we should force the vulkan drivers either, so I'd be against them. In other words, you can completely remove `depends`.